### PR TITLE
remove duplicated nodes in PV-tree

### PR DESCRIPTION
**Pull request by qlj-lijuan**
- Remove duplicated nodes in PV-tree
- delete unused fields in hierarchy.json

### CODE REVIEW

File: tools/pv_tree_generator/build_tree.py [hunk 1]
```diff
@@ -36,33 +36,32 @@ def build_tree_recursive(pos, level, pop_obs_spec, stat_vars, place_mapping, sho
...
         'title': text_format.format_title(prop_ui_node.text),
-        'type': 'Property',
+        'type': 'prop',
         'count': 0,
         'children': [],
         'sv_set': set(),
         'search_count': 0,
         'search_sv_set': set(),
-        'measuredProperty': prop_ui_node.mprop,
+        'mprop': prop_ui_node.mprop, #measuredProperty
```
Comment (shifucun) [inline tools/pv_tree_generator/build_tree.py:46]: remove this?

File: tools/pv_tree_generator/build_tree.py [hunk 2]
```diff
@@ -128,12 +125,9 @@ def build_tree(v, pop_obs_spec, stat_vars, place_mapping, show_all):
 
     #vertical as the root
     root = {
-        'show': 'yes',
-        'selected': 'no',
-        'expanded': 'no',
         'argString': 'top',
         'title': text_format.format_title(v),
-        'type': 'Property',
+        'type': 'prop', #property
```
Comment (shifucun) [inline tools/pv_tree_generator/build_tree.py:130]: can remove these comments

File: tools/pv_tree_generator/build_tree.py (hunk 1 above)
Comment (qlj-lijuan) [reply tools/pv_tree_generator/build_tree.py]: done.

File: tools/pv_tree_generator/build_tree.py (hunk 2 above)
Comment (qlj-lijuan) [reply tools/pv_tree_generator/build_tree.py]: done.

**Comment by qlj-lijuan** [comment]
> How big is the new json now?

The new json is 11M.

**Pull request merged**